ipaplatform/redhat/authconfig.py

```diff
@@ -187,9 +190,10 @@ def restore(self, path):
         if cfg:
             profile = cfg[0]
 
-            cmd = [
-                paths.AUTHSELECT, "select", profile,
-                " ".join(cfg[1]), "--force"]
+            cmd = [paths.AUTHSELECT, "select", profile]
+            if cfg[1]:
+                cmd.append(" ".join(cfg[1]))
```

Comment:
I think this is also wrong and should ``cmd.extend(cfg[1])``. authselect doesn't complain about the syntax but it behaves differently:

```
# authselect current --raw
sssd with-fingerprint with-silent-lastlog
# authselect select sssd 'with-fingerprint with-silent-lastlog' --force
Backup stored at /var/lib/authselect/backups/2018-11-28-07-54-05.0uFTrd
Profile "sssd" was selected.
The following nsswitch maps are overwritten by the profile:
- passwd
- group
- netgroup
- automount
- services

Make sure that SSSD service is configured and enabled. See SSSD documentation for more information.

# authselect select sssd with-fingerprint with-silent-lastlog --force
Backup stored at /var/lib/authselect/backups/2018-11-28-07-54-29.iNPMzl
Profile "sssd" was selected.
The following nsswitch maps are overwritten by the profile:
- passwd
- group
- netgroup
- automount
- services

Make sure that SSSD service is configured and enabled. See SSSD documentation for more information.
 
- with-fingerprint is selected, make sure fprintd service is configured and enabled

# 
```

Please notice that the second variant prints ``with-fingerprint is selected, make sure fprintd service is configured and enabled`` while the first does not. The second variant is equivalent to ``cmd.extend(cfg[1])``.